packages/pyright-internal/src/languageService/completionProviderUtils.ts

```diff
@@ -93,8 +94,19 @@ export function getTypeDetail(
                     }
                 }
             }
-
-            return name + ': ' + evaluator.printType(type, { expandTypeAlias });
+            // handle the case where the type is a function is assigned to a variable
```

Comment:
Nit: Comments in the pyright code base should start with a capital letter and end in a period if it is a complete sentence. Also, the comment doesn't make sense; please rephrase for clarity.